tests/Unit/Digitick/Sepa/TransferInformation/CustomerDirectDebitTransferInformationTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+namespace tests\Unit\Digitick\Sepa\TransferInformation;
+
+/***************************************************************
+ *  (c) 2015 networkteam GmbH - all rights reserved
```

Comment:
@jarnovanleeuwen I'm not that familiar with all those licenses but I think this conflicts with our `GNU Lesser General Public License v3.0` which we used for this project. Would you mind removing this?